deploy/v1alpha1_helmrelease_crd.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: helmrelease.k8s.wattpad.com
+spec:
+  group: k8s.wattpad.com
+  scope: Namespaced
+  version: v1alpha1
+  names:
+    kind: HelmRelease
+    listKind: HelmReleaseList
+    plural: helmreleases
+    singular: helmrelease
+    shortNames:
+      - rls
+  subresources:
+    status: {}
```

Comment:
This subresource will allow us to put the helm release state into the CRs for the API service to query.
https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#status-subresource